src/main/plugins/org.dita.html5/xsl/nav.xsl

```diff
@@ -238,6 +238,9 @@ See the accompanying LICENSE file for applicable license.
                         (not(@format) or @format = 'dita' or @format = 'ditamap') ">
           <xsl:value-of select="@copy-to"/>
         </xsl:when>
+        <xsl:when test="@chunk = 'to-content'">
+                    <xsl:value-of select="substring-before(@href,'#')"/>
```

Comment:
Actually, now that I reread it, probably not without refactoring other parts of the code.